code/streaming/Hacl.Streaming.Functor.fst

```diff
@@ -682,8 +708,10 @@ let split_at_last_rest_eq (#index : Type0) (c: block index)
   let sz = rest c i total_len in
   ()
 
+// This rlimit is a bit crazy, but this proof is not stable. It usually
+// goes through fast, but a high rlimit is a security against regression failures.
```

Comment:
famous last words ;-)

Reply:
although the fact that non-linear arithmetic is disabled is, indeed, a good guarantee that it'll be robust

Reply:
Yes, this doesn't protect against regression failures indeed... I mostly meant CI failures, for this exact PR ^^'
And I was quite surprised by the proof time variance, though I deactivated non-linear arithmetic - it would be worth investigating.